CHANGELOG.md

```diff
@@ -212,6 +212,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+* [\#11693](https://github.com/cosmos/cosmos-sdk/pull/11693) Add validation for gentx cmd.
```

Comment:
We need to move this to the unreleased section.

Reply:
Ah, it's part of Unreleased section already. May be will move it into improvements subsection.